reflect.go

```diff
@@ -529,24 +529,30 @@ func (t *Type) arrayKeywords(tags []string) {
 }
 
 func (t *Type) extraKeywords(tags []string) {
+	if t.Extras == nil {
+		t.Extras = map[string]interface{}{}
+	}
 	for _, tag := range tags {
 		nameValue := strings.Split(tag, "=")
 		if len(nameValue) == 2 {
-			t.setExtra(nameValue[0], nameValue[1])
+			name, value := nameValue[0], nameValue[1]
+			switch name {
+			case "minimum":
```

Comment:
This should be done in `setExtra()`.

Reply:
Thanks @alecthomas . I will move it there.